### PR TITLE
Chips docs are ready

### PR DESCRIPTION
**Pull request by DmitriiBabakaevTR**

### CODE REVIEW

File: source/includes/documentation/_vue_js_elements.md [hunk 1]
```diff
@@ -4083,72 +4083,95 @@ Available methods in Java JDI Light:
 
 [Here you can find Buttons tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/VuetifyButtonsTests.java).
 
-### 5.38 Chip
+### 5.38 Chips
 
-[Chip Vuetify documentation page](https://vuetifyjs.com/en/components/chips/)
-
-Chip is located in the following class:
+[Chips Vuetify documentation page](https://vuetifyjs.com/en/components/chips/)
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:4088]: should be v2. in the url :)

File: source/includes/documentation/_vue_js_elements.md [hunk 2]
```diff
@@ -4083,72 +4083,95 @@ Available methods in Java JDI Light:
...
+**getContent()** | Get chip content                              | UIElement
+**fontSize()** | Gets chip font size                           | String
+**borderColor()** | Gets chip border color                        | String
+**hasXSmallSize()** | Checks if the chip has x-small size           | boolean
+**hasSmallSize()** | Checks if the chip has small size             | boolean
+**hasDefaultSize()** | Checks if the chip has default size           | boolean
+**hasLargeSize()** | Checks if the chip has large size             | boolean
+**hasXLargeSize()** | Checks if the chip has x-large size           | boolean
+**isRemovable()** | Checks if the chip is revomable               | boolean
+**isPill()** | Checks if the chip is pill                      | boolean
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:4174]: we agreed to add info about implemented interfaces like 'mplements HasClick, HasColor, HasIcon, HasImage,
        HasTheme, IsOutlined, HasMeasurement'

File: source/includes/documentation/_vue_js_elements.md [hunk 3]
```diff
@@ -4083,72 +4083,95 @@ Available methods in Java JDI Light:
...
 
 For examples of usage see: [JDI Vuetify Chip tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/ChipsTests.java).
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:4177]: 06 03 23 agreed to change the link to master https://github.com/jdi-testing/jdi-light/blob/master/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/ChipsTests.java

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 1 above)
Comment (DmitriiBabakaevTR) [reply source/includes/documentation/_vue_js_elements.md:4088]: fixed

File: source/includes/documentation/_vue_js_elements.md (hunk 3 above)
Comment (DmitriiBabakaevTR) [reply source/includes/documentation/_vue_js_elements.md:4177]: fixed

File: source/includes/documentation/_vue_js_elements.md (hunk 2 above)
Comment (DmitriiBabakaevTR) [reply source/includes/documentation/_vue_js_elements.md:4262]: fixed

**Pull request merged**